<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>about_dialog</class>
 <widget class="QDialog" name="about_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>关于 CipherManagerGUI</string>
  </property>
  <layout class="QGridLayout" name="grid_layout">
   <item row="0" column="1">
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>加载失败</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="about_cm_label">
     <property name="text">
      <string>CipherManager版本：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="about_label">
     <property name="text">
      <string>CipherManagerGUI版本：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="cm_version_label">
     <property name="text">
      <string>加载失败</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="license_group_box">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QGridLayout" name="license_group_box_grid_layout">
      <item row="1" column="0">
       <widget class="QTextEdit" name="license_text_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>about_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
